app/models/event.rb

```diff
@@ -240,5 +241,11 @@ def user_cannot_be_inactive
 
     errors.add(:base, "You need an active subscription to capture events with your live API key")
   end
+
+  def property_cannot_be_all
+    return unless properties.keys.include?("All") || properties.keys.include?("all") 
+
+    errors.add(:properties, "'All' and 'all' are reserved property names for Fugu and can't be used")
```

Comment:
should I use the value the user entered? or is it enough to just mention both "All" and "all"?

Reply:
I would make it even more general. Something like:
`"You've used a property name that's reserved by Fugu (such as 'all'). Learn more about property constraints in the Fugu docs: https://docs.fugu.lol"